kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/OperatorMain.java

```diff
@@ -31,22 +36,23 @@
 public class OperatorMain {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(OperatorMain.class);
-    private MeterRegistry registry;
     private final Operator operator;
+    private final HttpServer managementServer;
 
-    public OperatorMain() {
-        this(null);
+    public OperatorMain() throws IOException {
+        this(null, HttpServer.create(new InetSocketAddress("0.0.0.0", 8080), 10));
     }
 
-    public OperatorMain(@Nullable KubernetesClient kubeClient) {
-        final MicrometerMetrics metrics = enablePrometheusMetrics();
+    public OperatorMain(@Nullable KubernetesClient kubeClient, @NonNull HttpServer managementServer) {
```

Comment:
should this ctor have a VisibleForTesting annotation?

Reply:
done and reduced the visibility.